### ISSUE TITLE
Alt text / labeling issues in image cropping UI

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#5309]
### Issue Summary

Related: #5291. Spotted while reviewing #5292. There are three accessibility issues picked up by Axe in the image cropping UI:

<img width="937" alt="image-cropping" src="https://user-images.githubusercontent.com/877585/58026571-aafd5b00-7b0e-11e9-8ff4-2d43bd843cbb.png">

- jcrop generates its own `img` element. That element does not even have an alt text, so the image `src` is read out loud.
- The image tag rendered server-side uses the 
`title` attribute for the alt text. While this might be appropriate on the site’s front-end, here the image is displayed in the admin, to be edited, so the `title` text does not work as an `alt` in that context. Relevant code: https://github.com/wagtail/wagtail/blob/70a749b10a1b0c4dac527da28e54cbb3581a774e/wagtail/images/templates/wagtailimages/images/edit.html#L67
- jcrop injects its own `radio` element, which is flagged as not having a label

### Steps to Reproduce

1. Go to the image editing UI on any Wagtail admin, `/admin/images/52/`
2. Navigate to the cropping UI with a screen reader

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: yes

### Technical details

* Wagtail version: v2.6a0

---

I think the simplest solution for this would be to make sure the image created with jcrop has an empty `alt=""` attribute, and same for our server-generated image. This might need further consideration for us to design a more screen-reader-friendly solution.

**Comment by Luzzzi** [comment]
Hi @thibaudcolas ! Can I work on this ? 

I consider adding a `label` to the `input` and `name ` in `input`, something like this : 
`<label for="jcrop" classname="visuallyhidden">Crooping tool</label>
<input name="jcrop">`
in https://github.com/wagtail/wagtail/blob/8b6b0425de87a802227b11ee8247c975e912ba83/wagtail/images/static_src/wagtailimages/js/focal-point-chooser.js#L32

What do you think ?
